src/services/message.service.js

```diff
@@ -0,0 +1,29 @@
+import { ApiError } from '../exceptions/ApiError.js';
+import { Message } from '../models/messages.js';
+
+const createMessage = async (author, text, roomId) => {
+  if (!author) {
+    throw ApiError.unauthorized();
+  }
+
+  if (!text) {
+    throw ApiError.badRequest('Message is required');
+  }
+
+  const room = await join(roomId);
```

Comment:
The `join` function is called here but is not defined or imported in this file. Ensure that you import the `join` function from the appropriate module or define it within this file.